docs/docusaurus.config.ts

```diff
@@ -98,7 +100,7 @@ const config: Config = {
           items: [
             {
               label: 'Overview',
-              to: '/docs/',
+              to: '/pgov-cms/',
```

Comment:
@vauxia this all looks good, but wondering:  above you update `/` to `/pgov-cms/` and here you update `/docs/`... is this expected?